Tests/TestingApps/PaywallsTester/PaywallsTester/Data/ApplicationData.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  ApplicationData.swift
+//
+//
+//  Created by Nacho Soto on 12/11/23.
+//
+
+import Foundation
+
+import OSLog
+
+@Observable
+final class ApplicationData {
+
+    private(set) var authentication: Authentication = .unknown {
```

Comment:
`authentication` feels like an unintuitive name, like, I would usually call this something like `authenticationStatus` but that might be redundant

Reply:
Agree and done!